.github/workflows/build_and_test.yml

```diff
@@ -89,11 +89,14 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
+    - run: sudo apt install gcc-multilib
+
     - name: Install Rust
       uses: actions-rs/toolchain@v1
       with:
         profile: minimal
         components: rustfmt, clippy
+        target: i686-unknown-linux-gnu
```

Comment:
Just checking, but this is an additional target, the default is still x86_64?

Reply:
Correct (and verified in the build log by seeing the 32-bit tests fail, oops).